packages/api-client-core/src/GadgetConnection.ts

```diff
@@ -437,7 +437,7 @@ export class GadgetConnection {
     return client;
   }
 
-  private newSubscriptionClient(overrides: GadgetSubscriptionClientOptions) {
+  newSubscriptionClient(overrides?: GadgetSubscriptionClientOptions) {
```

Comment:
Did you mean to make `overrides` required?

Reply:
I think that was an accident from before -- the whole code deals with it being optional already, and i think overrides shouldn't be mandatory